_includes/layouts/docs.html

```diff
@@ -37,12 +37,6 @@ <h2>Table of Contents</h2>
           <div class="grid-col flex-4"></div>
           <div class="grid-col flex-1">
             <p>
-              <a
```

Comment:
Let's expand this on all of these template to get rid of the entire row since it's not being used for anything. So let's chop all of this, beginning with the `<div class="grid-row">`:
```
        <div class="grid-row">
          <div class="grid-col flex-4"></div>
          <div class="grid-col flex-1">
            <p>
              <a
                href="{{ site.github_url }}/edit/{{ site.github_branch }}/{{ page.relative_path }}"
                class="usa-link--external"
                target="_blank"
                >Suggest edits</a
              >
            </p>
          </div>
        </div>
```